src/Logging/Logger.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Logging;
+
+use Google\Cloud\Logging\Connection\ConnectionInterface;
+
+class Logger
+{
+    /**
+     * @var ConnectionInterface Represents a connection to Cloud Logging.
+     */
+    private $connection;
+
+    /**
+     * @var string The identity of the logger.
+     */
+    private $identity;
+
+    /**
+     * @param ConnectionInterface $connection Represents a connection to Cloud
+     *        Logging.
+     * @param string $name The name of the log to write entries to.
+     * @param string $projectId The project's ID.
+     */
+    public function __construct(ConnectionInterface $connection, $name, $projectId)
+    {
+
+    }
+
+    /**
+     * Deletes this log and all its log entries. The log will reappear if it
+     * receives new entries.
+     *
+     * @param array $options Configuration options.
+     */
+    public function delete(array $options = [])
+    {
+
+    }
+
+    /**
+     * Creates an entry which is to be written to the log.
+     *
+     * Example:
+     * ```
+     * // Create an entry with a key/value set of data
+     * $entry = $log->entry(
+     *     ['user' => 'calvin'],
+     *     [
+     *         'type' => 'gcs_bucket',
+     *         'labels' => [
+     *             'bucket_name' => 'my-bucket'
+     *         ]
+     *     ]
+     * );
+     * ```
+     *
+     * ```
+     * // Create an entry with a string
+     * $entry = $log->entry('my message', [
+     *    'type' => 'gcs_bucket',
+     *    'labels' => [
+     *        'bucket_name' => 'my-bucket'
+     *    ]
+     * ]);
+     * ```
+     *
+     * @param array|string $data The data to log.
```

Comment:
There are examples in the docblock. Would you like to see more?

If string it will send the payload as 'text' if an array it will send it as 'json' - https://cloud.google.com/logging/docs/api/ref_v2beta1/rest/v2beta1/LogEntry